doc/rados/troubleshooting/cpu-profiling.rst

```diff
@@ -9,59 +9,72 @@ you can profile Ceph's CPU usage. See `Installing Oprofile`_ for details.
 Initializing oprofile
 =====================
 
-The first time you use ``oprofile`` you need to initialize it. Locate the
-``vmlinux`` image corresponding to the kernel you are now running. :: 
+``oprofile`` must be initalized the first time it is used. Locate the
```

Comment:
I never know whether to capitalize such cases

Reply:
I have a strong intuition that the command or program name should be kept as it is, as here. Cole fights like hell to find a way to put a word or phrase before commands or program names like this.

Reply:
That's my usual practice as well.

Reply:
>The ``oprofile`` system must be initialized ...